boards/chronos/include/periph_conf.h

```diff
@@ -26,9 +26,16 @@ extern "C" {
 #endif
 
 /**
- * @brief   Clock configuration
+* @todo
+*
+* @{
+* Move all clock configuration code here from the board.h
+* @}
+*/
+/**
+ * @name   Clock configuration
  *
- * @todo    Move all clock configuration code here from the board.h
```

Comment:
I think the TODO needs to be put to a *documented* variable. I'm trying something out.